tests/test_ops/test_contour_expand.py

```diff
@@ -0,0 +1,42 @@
+import numpy as np
+
+
+def test_contour_expand():
+    from mmcv.ops import contour_expand
+
+    np_internal_kernel_label = np.array([[0, 0, 0, 0, 0, 0, 0, 0, 0, 0],
```

Comment:
If the op supports both ndarray and tensor, we'd better also add test cases of torch tensors and check the consistency between ndarray and tensor.